themes/src/main/resources/theme/base/account/account.ftl

```diff
@@ -59,7 +59,7 @@
         <div class="form-group">
             <div id="kc-form-buttons" class="col-md-offset-2 col-md-10 submit">
                 <div class="">
-                    <#if url.referrerURI??><a href="${url.referrerURI}">${msg("backToApplication")?no_esc}/a></#if>
+                    <#if url.referrerURI??><a href="${url.referrerURI}">${kcSanitize(msg("backToApplication")?no_esc)}</a></#if>
```

Comment:
I take it there's no way to be able to do ?kc_sanitize instead?

Reply:
That's correct.  I wanted to do that but FreeMarker doesn't allow it.